app/controllers/api_controller.rb

```diff
@@ -244,18 +244,18 @@ def body_request_events
       render template: "api/request_events", formats: [:atom], layout: false
     elsif feed_type == "json"
       @event_data = []
-      @events.each do |event|
+      @events.each do |e|
```

Comment:
Ah sorry if I wasn't clear, multi line `do`/`end` blocks shouldn't use a single lowercase letter. That was only for single line `{`/`}` blocks.

Reply:
Oh, that makes sense!